### PR TITLE
Fixed #30 Add explanation about error in Python scripts.

### LINKED ISSUE #30: Add explanation about error in Python scripts
Add some explanations saying that errors identified in the import lines are "normal".

This explanation should be added in the Readme of the repo (at the end), and in the User Manual (section Run a Python script)

### PR DESCRIPTION
**Pull request by ylussaud**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -139,3 +139,21 @@ When you will open your fist Python file, you will be prompted by PyDev to defin
...
+
+- some import directives from EASE are not recognized by PyDev
+
+![Python4Capella project content](https://raw.githubusercontent.com/labs4capella/python4capella/master/README/error_import.png)
+
+- some methods defined in EASE modules are not recognized by PyDev
+
+![Python4Capella project content](https://raw.githubusercontent.com/labs4capella/python4capella/master/README/error_method.png)
+
+- some other errors can happen when accessing varaibles defined by EASE
```
Comment (mbats) [inline README.md:155]: variables

File: README.md [hunk 2]
```diff
@@ -139,3 +139,21 @@ When you will open your fist Python file, you will be prompted by PyDev to defin
 You can validate the dialog by clicking the OK button.
 
 You can refere to the plugin documentation for usage reference. The documentation is accessible by using the Help > Help contents menu from capella, then on the right tree, you can select Python4Capella.
+
+# Errors in provided scripts
+
+If you open provided scripts, you will notice some errors in them. Those errors can be ignored. They are related to the use of EASE that allows access to Java objects with the PyDev editor:
```
Comment (mbats) [inline README.md:145]: Explain what is "EASE"

File: README.md [hunk 3]
```diff
@@ -139,3 +139,21 @@ When you will open your fist Python file, you will be prompted by PyDev to defin
...
+
+- some import directives from EASE are not recognized by PyDev
```
Comment (mbats) [inline README.md:147]: Make "import directives from EASE" bold.

File: README.md [hunk 4]
```diff
@@ -139,3 +139,21 @@ When you will open your fist Python file, you will be prompted by PyDev to defin
...
+
+![Python4Capella project content](https://raw.githubusercontent.com/labs4capella/python4capella/master/README/error_import.png)
+
+- some methods defined in EASE modules are not recognized by PyDev
```
Comment (mbats) [inline README.md:151]: Make "methods" bold and give some more examples

File: README.md (hunk 1 above)
Comment (mbats) [reply README.md:155]: Make "variables defined by EASE" in bold

File: README.md (hunk 2 above)
Comment (mbats) [reply README.md:145]: Add an explanation that even if these errors exist, the scripts are valid.

**Review by mbats — APPROVED** [review]

File: README.md (hunk 4 above)
Comment (ylussaud) [reply README.md:151]: done

File: README.md (hunk 3 above)
Comment (ylussaud) [reply README.md:147]: done

**Pull request merged**